R/utils_render_rtf_redo.R

```diff
@@ -240,6 +240,134 @@ rtf_table <- function(rows) {
   rtf_raw(paste(unlist(rows), collapse = ""))
 }
 
+# This is the assumed width of the area within page
+# margins in a default Word document
+standard_width_twips <- 9468L
+
+twip_factors <-
+  c(
+    `in` = 1440, `pt` = 20, `px` = 15,
+    `cm` = 566.9291, `mm` = 56.69291, `tw` = 1
+  )
+
+parse_length_str <- function(lengths_vec) {
+
+  if (length(lengths_vec) == 1 && lengths_vec == "auto") {
```

Comment:
Instead of having `"auto"` be understood by `parse_length_str()`, handle `"auto"` in the caller because it's impossible to interpret it here (not enough context).

Reply:
Now changed.